playbooks/cloud-provider-openstack-acceptance-test-lb-octavia/run.yaml

```diff
@@ -1,124 +1,144 @@
 - hosts: all
   become: yes
   roles:
-    - config-golang
+    - config-docker-machine
     - export-cloud-openrc
-    - install-k8s
   tasks:
-    - name: Run lbass octavia acceptance tests with cloud-provider-openstack
+    - name: Login dockerhub
+      shell: docker login -u {{ dockerhub.username }} -p {{ dockerhub.password }}
+      no_log: yes
+
+    - name: Install kubectl and jq
       shell:
         cmd: |
-          set -x
-          set -e
-          set -o pipefail
+          curl -LO https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/linux/amd64/kubectl
+          chmod +x ./kubectl
+          mv ./kubectl /usr/local/bin/kubectl
+          mkdir ~/.kube
+          cat << EOF >> ~/.kube/config
+          {{ kubeconfig.content }}
```

Comment:
Seems here, the kubeconfig secret is lost, please wait for the https://github.com/theopenlab/openlab-zuul-jobs/pull/793 merged.

Reply:
And need to inject kubeconfig in the job definition.

Reply:
https://github.com/theopenlab/openlab-zuul-jobs/blob/master/zuul.d/jobs.yaml#L822  . Once all of done, we have a try.

Reply:
https://github.com/theopenlab/openlab-zuul-jobs/blob/master/zuul.d/jobs.yaml#L822  is merged.